docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.5
+LABEL maintainer Jan Koppe <post@jankoppe.de>
+
+ARG PUMPIO__GUID=1337
```

Comment:
Yes, the choide of UID/GUID is  really arbitrary in this case. I included it as a build ARG in case the file permissions etc. in deployment differ from the default user. This makes running it in production a bit more comfortable.

Any proposals for a system user UID/GUID?